### ISSUE TITLE
Pleas update docs for IEncryptionProvider

### ISSUE THREAD

**Issue by Polemus** [Eastrall/EntityFrameworkCore.DataEncryption#37]

**Comment by Eastrall** [comment]
The encryption provider documentation will be updated in v4 since the system has changed and is now much easier to setup.
The interface is simplier, it just takes a `byte[]` input and needs to return the encrypted/decrypted data as a `byte[]`:
```csharp
public interface IEncryptionProvider
{
    byte[] Encrypt(byte[] input);
    byte[] Decrypt(byte[] input);
}
```
Please wait a little bit for v4 release, I should release it at the same time as .NET 7 release.

**Comment by Eastrall** [comment]
Shipped in 4.0.0
